### ISSUE TITLE
Wrong multiplication in MNIST beginner tutorial

### ISSUE THREAD

**Issue by urinieto** [tensorflow/tensorflow#146]
(Not sure if this is the right place to report a problem with [this tutorial](http://tensorflow.org/tutorials/mnist/beginners/index.md).)

I believe that the explanation of matrix multiplication in [this image](http://api.tensorflow.org/system/image/body/1707/softmax-regression-scalarequation.png) is wrong.

It should be W_1,1 x_1 + W_1,2 x_2 + W_1,3 x_3... (i.e., x should increase per row, not per column).

**Comment by vrv** [comment]
This is fixed at HEAD, hasn't yet been pushed to the website.  Thanks for the report!

De-duping with https://github.com/tensorflow/tensorflow/issues/78
